website/docs/en/blog/announcing-1-0.mdx

```diff
@@ -0,0 +1,209 @@
+- **Release Rsdoctor 1.0.** This release will improve support for Vue and provide report formats for CI/CD.
+- **Develop Rslib 0.x.** Rslib is a library building tool based on Rsbuild. See [Rslib repository](https://github.com/web-infra-dev/rslib) for more details.
+- **Develop Rspress 2.0.** It will be based on React 19 and will improve some of the API designs. See [Rspress v2.0 planning](https://github.com/web-infra-dev/rspress/discussions/1105) for more details.
+
+Here are some key features we plan to support in Rspack 1.x:
+
+### Faster HMR
+
+Rspack can currently meet the performance requirements for most projects, but there is still significant room for performance optimization. During development, Rspack has already achieved nearly constant level incremental builds during the make phase. However, in the seal phase, some computations can still slow down as the project scales. Rspack will incrementally optimize the computations in the seal phase to keep the HMR time at a constant level.
+
+### Portable cache
+
+The evolution path of Rspack's caching capabilities follows a sequential implementation of memory cache, persistent cache, and portable cache. Currently, Rspack has implemented a memory cache that provides excellent HMR performance. The next step is to implement a persistent cache based on this foundation to address long cold startup times for large projects and to functionally align with webpack.
+
+After that, we plan to continue implementing the portable cache. This means that Rspack's build cache will not only be persistent, but also portable across environments and machines. This will help teams make better use of the cache and lay the groundwork for distributed builds.
+
+### TypeScript-based optimization
+
+Currently, when Rspack processes TypeScript modules, it first converts them to JavaScript through a loader before further processing. This provides flexibility but also hinders further optimization of the build output. For example, developers need to use `enum` instead of `const enum`, but `enum` is difficult to optimize as a constant. In the future, we plan to treat TypeScript as a first-class citizen in Rspack, leveraging TypeScript's static information to provide more advanced compile-time optimization of the build output (such as [type-based property renaming](https://github.com/google/closure-compiler/wiki/Type-Based-Property-Renaming)).
+
+### Stable Rust API
+
+Currently, higher-level tools can use the JS API to integrate Rspack, which provides good extensibility. However, the communication overhead between Rust and JavaScript that limits the performance of Rspack. We also provide the SWC Wasm plugin to support extensions, but its performance is still slower than native languages.To provide higher-level tools with more flexible integration options and better performance, we plan to expose Rspack's Rust API for integration.
```

Comment:
When you are saying higher-level tools, maybe name same examples like: 

```suggestion
Currently, higher-level tools (Rsbuild or build tools developed on top of Rspack) can use the JS API to integrate Rspack, which provides good extensibility. However, the communication overhead between Rust and JavaScript that limits the performance of Rspack. We also provide SWC Wasm plugin to support extensions, but its performance is still slower than native languages. To provide higher-level tools with more flexible integration options and better performance, we plan to expose Rspack's Rust API for integration.
```

Reply:
Are you referring to SWC WASM plugin as in this one ? https://rspack.dev/plugins/webpack/enable-wasm-loading-plugin

or this one ? https://swc.rs/docs/usage/wasm

Reply:
The SWC one

Reply:
```suggestion
Currently, higher-level tools can use the JS API to integrate Rspack, which provides good extensibility. However, the communication overhead between Rust and JavaScript that limits the performance of Rspack. We also provide the [SWC Wasm plugin](https://swc.rs/docs/usage/wasm) to support extensions, but its performance is still slower than native languages.To provide higher-level tools with more flexible integration options and better performance, we plan to expose Rspack's Rust API for integration.
```

Reply:
done